### DISCUSSION TITLE
CORS Error- how can I get a Access-Control-Allow-Origin header?

### DISCUSSION THREAD

**Discussion by alessiogmonti** [community/community#12209] [Other Feature Feedback, Questions, & Ideas]
I am receiving these two errors 
`Access to XMLHttpRequest at 'linktomyscript' from origin 'mywebpage' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
`

`
POST 'linktomyscript' net::ERR_FAILED 200
`

and this warning: 
`Error with Permissions-Policy header: Unrecognized feature: 'interest-cohort'.`

I am trying to POST a form submission to a google sheets script on my github pages site. 
How can fix?

**Comment by Gadem** [comment]
Well... i want to know to.

**Reply by Randle-Lanre** [reply]
@alessiogmonti if you allow all headers, does it work ? or you get the same error ?

**Reply by alessiogmonti** [reply]
Hey @Randle-Lanre, I don't know how to do that. I did read that if a Pages site is public, all CORS are allowed by default. If like me you just care about getting form submission working, you can use https://un-static.com/ which is a great tool, though I'm not sure if they store the data somewhere.
